### DISCUSSION TITLE
Sound doesn't work on Arch linux

### DISCUSSION THREAD

**Discussion by tbvns** [ppy/osu#21470] [Q&A]
### Type

Game behaviour

### Bug description

When i launch osu, I just have this error:
(in consol): 
ALSA lib pcm_dmix.c:999:(snd_pcm_dmix_open) unable to open slave
ALSA lib pcm_dmix.c:999:(snd_pcm_dmix_open) unable to open slave
ALSA lib pcm_dmix.c:999:(snd_pcm_dmix_open) unable to open slave
ALSA lib pcm_dmix.c:999:(snd_pcm_dmix_open) unable to open slave
ALSA lib pcm_dmix.c:999:(snd_pcm_dmix_open) unable to open slave
kf.service.services: KApplicationTrader: mimeType "x-scheme-handler/file" not found
Fontconfig error: "/etc/fonts/local.conf", line 1: XML declaration not well-formed
Fontconfig error: "/etc/fonts/local.conf", line 1: XML declaration not well-formed
ALSA lib pcm_dmix.c:999:(snd_pcm_dmix_open) unable to open slave

### Screenshots or videos

(in game):
![image](https://user-images.githubusercontent.com/69420062/204792016-1970c029-22b5-4bbd-a763-7aa53b711f0a.png)

### Version

2022.1123.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/10122974/database.log)
[input.log](https://github.com/ppy/osu/files/10122975/input.log)
[network.log](https://github.com/ppy/osu/files/10122977/network.log)
[performance.log](https://github.com/ppy/osu/files/10122978/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10122979/runtime.log)

**Comment by Axilotl17** [comment]
Hmm I use arch w/KDE as well and I don't have any issues... do you use Pipewire, Pulseaudio, or any other audio framework? 


![image](https://user-images.githubusercontent.com/78441708/205221054-56bc744d-a83f-4e99-bd07-10455b9a09a0.png)
also try navigating here in settings and messing around with it. It isn't always set to the right device.

**Reply by itsskyballs** [reply]
check the sound card

**Reply by itsskyballs** [reply]
i trying to use MediaTek sound card and it works
but i think pipewire didn't support

**Comment by tbvns** [comment]
I installed pulse audio and it work
ty

**Reply by Axilotl17** [reply]
I recommend pipewire, but okay.

**Comment by artem** [comment]
Same issue with Pipewire on Kubuntu 22.10. The workaround is to close all media application and browsers.

Seems that for some reason Osu tries to gain exclusive access to sound card (and fails if there are other active sound clients, like Firefox)

**Reply by artem** [reply]
Installing `pipewire-alsa` resolved the issue

**Reply by itsskyballs** [reply]
i think arch linux cannot play the audio
check the audio setting if the pipewire cannot hear the lazer sound

**Comment by walksanatora** [comment]
I use PulseAudio and was able to fix this by setting the output to the PulseAudio server

**Comment by KitsuneNoNatsu** [comment]
Hello, recently installed lazer, none of these aforementioned fixes work on my end, but I have an extra issue
If I select my headphones as the audio output, I can hear audio, but then it hogs my audio, and no other audio works

**Reply by walksanatora** [reply]
do you have Pulse/ALSA installed
and have you made sure they are running

**Comment by KitsuneNoNatsu** [comment]
Yes and yes, OSU stable works fine, every other game works fine, this is the first time I've encountered this issue after using Arch for 7 months

Get BlueMail for Android<https://bluemail.me>
On Apr 17, 2023, at 08:10, Walker Fowlkes ***@***.******@***.***>> wrote:

do you have Pulse/ALSA installed
and have you made sure they are running

**Reply by walksanatora** [reply]
when you go to `output device` what is listed
did you also make sure to try and scroll down

**Reply by walksanatora** [reply]
(I also use Arch so idk what problem you are having as it Works on My Machine:tm:)
